src/fragments/lib/auth/js/switch-auth.mdx

```diff
@@ -130,4 +217,4 @@ export const handler = async (event, context) => {
 
   return event;
 };
-```
+
```

Comment:
Why is this removed?

Reply:
Added them back in @elorzafe.  FYI for anyone that doesn't know, it was fun trying to figure out how.  Apparently the suggestion backticks must be greater than the code example backticks
https://github.com/jonschlinkert/remarkable/issues/146#issuecomment-85539428

Reply:
Thanks for catching this!